preprocessors/openstreetmap/osm_service.py

```diff
@@ -604,7 +611,9 @@ def allot_intersection(processed_OSM_data, inters_rec_up
                     if id1 != id2:  # compare unique street only
                         # check if a node represents an intersection
                         if nodes[i] == intersection_nodes[items]:
-                            nodes[i]["cat"] = "intersection"
```

Comment:
This was changed based on the discussion we had to include ```ids``` of intersecting streets instead of indicating if there is an intersection or not.

Reply:
Understandable, but why is it in *this* pull request?